diff --git a/app/app.vue b/app/app.vue
index cb502028d08e2477d6891f15fb86818e99339a56..55095ff5315044f479b776cb45ec6e562b43caa3 100644
--- a/app/app.vue
+++ b/app/app.vue
@@ -1,6 +1,6 @@
 <script setup lang="ts">
 import type { ContentNavigationItem, PageCollections } from '@nuxt/content'
-import * as nuxtUiLocales from '@nuxt/ui/locale'
+import { en } from '@nuxt/ui/locale'
 import { getLocaleKey } from '../utils/locale'
 import { transformNavigation } from './utils/navigation'
 import { useDocusShortcuts } from './composables/useDocusShortcuts'
@@ -13,6 +13,7 @@ const site = useSiteConfig()
 const { locale, locales, isEnabled, switchLocalePath } = useDocusI18n()
 const { isEnabled: isAssistantEnabled } = useAssistant()
 
+const nuxtUiLocales = { en }
 const nuxtUiLocale = computed(() => nuxtUiLocales[getLocaleKey(locale.value) as keyof typeof nuxtUiLocales] || nuxtUiLocales.en)
 const lang = computed(() => nuxtUiLocale.value.code)
 const dir = computed(() => nuxtUiLocale.value.dir)
diff --git a/app/error.vue b/app/error.vue
index c3c359c427bb3670e872c94735de8f06e9c506fe..5845a332c67e4649478741a43dad874dd14cfa9c 100644
--- a/app/error.vue
+++ b/app/error.vue
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import type { NuxtError } from '#app'
 import type { ContentNavigationItem, PageCollections } from '@nuxt/content'
-import * as nuxtUiLocales from '@nuxt/ui/locale'
+import { en } from '@nuxt/ui/locale'
 import { getLocaleKey } from '../utils/locale'
 import { transformNavigation } from './utils/navigation'
 
@@ -11,6 +11,7 @@ const props = defineProps<{
 
 const { locale, locales, isEnabled, t, switchLocalePath } = useDocusI18n()
 
+const nuxtUiLocales = { en }
 const nuxtUiLocale = computed(() => nuxtUiLocales[getLocaleKey(locale.value) as keyof typeof nuxtUiLocales] || nuxtUiLocales.en)
 const lang = computed(() => nuxtUiLocale.value.code)
 const dir = computed(() => nuxtUiLocale.value.dir)
